keyboards/avalanche/v1/keymaps/default/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 Vitaly Volkov (@vlkv)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define USE_SERIAL
```

Comment:
This is the default behavior and isn't technically needed.  Nor is the rest of the fine, in that case.